### PR TITLE
Migrate gatsby config files to TS

### LINKED ISSUE #91: Config files written in ts

### PR DESCRIPTION
**Pull request by triszt4n**
Closes #91

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +0,0 @@
-gatsby-*.js
```
Comment (triszt4n) [inline .eslintignore:1]: This way config files can be supervised as well

File: config/gatsby-config.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    baseUrl: 'https://kir-dev.sch.bme.hu',
+    lang: 'hu',
+    title: 'Kir-Dev',
+    titleTemplate: '%s | Kir-Dev',
+    description: 'A Kir-Dev kör blogja',
+    author: 'kir-dev',
+    image: '/favicon.png',
+    keywords: ['gatsbyjs', 'gatsby', 'javascript', 'ruby', 'rails', 'nodejs', 'typescript', 'community', 'simonyi'],
+    robots: 'index, follow'
+  },
```
Comment (triszt4n) [inline config/gatsby-config.ts:19]: Added a bit more site metadata

File: config/gatsby-node.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      name: `slug`,
+      value: `/${layout}${slug}`
+    })
+    createNodeField({
+      node,
+      name: `layout`,
+      value: layout
+    })
+  }
+}
```
Comment (triszt4n) [inline config/gatsby-node.ts:27]: Fresh from gatsby tutorial: https://www.gatsbyjs.com/docs/creating-slugs-for-pages/

**Review by OmTheTurtle — DISMISSED** [review]

**Comment by triszt4n** [comment]
Sry I had to rebase because of the chakra font switches

**Review by OmTheTurtle — APPROVED** [review]

**Pull request merged**